### ISSUE TITLE
invalid byte sequence

### ISSUE THREAD

**Issue by scarver2** [glebm/i18n-tasks#141]
I'm encountering issues running I18n-tasks. My online research has not uncovered any solutions.

Here are my output logs.
https://gist.github.com/scarver2/f136b23f4b34606f18a1

**Comment by lenart** [comment]
I had the same problem just now. Because I needed a quick solution I opened a file causing the error `~/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/i18n-tasks-0.8.0/lib/i18n/tasks/scanners/pattern_scanner.rb`. I first output the paths to see which file blows up the scanning. In my case it was an otf font in `app/assets/fonts/` so I edited the `scan_file` method to skip files in my app/assets.

```ruby
    def scan_file(path, opts = {})
      return [] if path.match /app\/assets\//
      # ... rest of the code omitted
   end
```

This is not a solution but at the moment I cannot dig deeper. Hope it helps.

**Comment by glebm** [comment]
I will release a new version with `.otf` excluded by default and a better error message. Meanwhile, put this in `config/i18n-tasks.yml` if you don't want to scan assets:

```yml
search:
  exclude:
    - 'app/assets/**'
```

**Comment by glebm** [comment]
Should be fixed in v0.8.1, please update and set `search.exclude` to:

```yml
search:
  exclude:
    - app/assets/images
    - app/assets/fonts
```

**Comment by scarver2** [comment]
Thank you. Ultimately, I discovered it was a SWF file in my assets that was throwing the exception. Adding the search exclusions to the YAML file resolved this.
